aws/dsba/README.md

```diff
@@ -89,7 +89,15 @@ Description of the different flows.
 
 ### Onboarding
 
-TODO
+* The user initiates the login at the onboarding portal. A QR code is displayed.
+* The user scans the QR code with its mobile wallet. The QR code contains a URL of the verifier to initiate the login.
```

Comment:
Stefan extended the login description a bit. Either copy from there or put a reference to align with the description: https://github.com/FIWARE-Ops/fiware-gitops/blob/master/aws/dsba/README.md#login